### PR TITLE
EclecticIQ -  2fa update

### PR DESCRIPTION
**Pull request by sasha-eiq**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added 2FA auth compatibility. Starting with EclecticIQ version 2.7 user can use either password or token for authentication.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Integrations/EclecticIQ/EclecticIQ.py [hunk 1]
```diff
@@ -135,18 +136,24 @@ def login():
...
+        response = http_request('POST', cmd_url, cmd_json=cmd_json)
+        if 'token' in response:
+            token = response['token']
+            HEADERS['Authorization'] = 'Bearer {}'.format(token)
+        else:
+            return_error('Failed to retrieve token')
     else:
-        return_error('Failed to retrieve token')
-    HEADERS['Authorization'] = 'Bearer {}'.format(token)
+        HEADERS['Authorization'] = 'Bearer {}'.format(PASSWORD)
```
Comment (anara123) [inline Integrations/EclecticIQ/EclecticIQ.py]: what will be the username in this case? the same username?

**Comment by anara123** [comment]
@sasha-eiq Also add description on how to create/generate/get this token in `EclecticIQ_description.md`
In that file we explain how to configure the instance of `EclecticIQ`. So you can explain that the integration supports API_TOKEN too, and that token can be set in the field of `Password`

File: Integrations/EclecticIQ/EclecticIQ.py (hunk 1 above)
Comment (sasha-eiq) [reply Integrations/EclecticIQ/EclecticIQ.py]: yes, the same username.

**Comment by anara123** [comment]
@kirbles19 Please review the PR

**Comment by anara123** [comment]
@sasha-eiq The PR is approved. Our technical writer will review the PR and we will merge it right after it

**Comment by kirbles19** [comment]
@anara123 Done.

**Pull request merged**